[2021-07-18 05:05:07,233] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_2 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 05:05:07,614] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_2 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 05:05:07,614] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 05:05:07,614] {taskinstance.py:1088} INFO - Starting attempt 6 of 6
[2021-07-18 05:05:07,614] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 05:05:07,742] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): processing_tasks.task_2> on 2021-07-17T03:39:23.918881+00:00
[2021-07-18 05:05:07,751] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'processing_tasks.task_2', '2021-07-17T03:39:23.918881+00:00', '--job-id', '47', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpj2iogjsk', '--error-file', '/tmp/tmp_wc3oy7v']
[2021-07-18 05:05:07,752] {standard_task_runner.py:52} INFO - Started process 1839 to run task
[2021-07-18 05:05:07,753] {standard_task_runner.py:77} INFO - Job 47: Subtask processing_tasks.task_2
[2021-07-18 05:05:08,971] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.processing_tasks.task_2 2021-07-17T03:39:23.918881+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-18 05:05:10,464] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-07-17T03:39:23.918881+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-17T03:39:23.918881+00:00
[2021-07-18 05:05:10,464] {logging_mixin.py:104} INFO - This is task 2 function
[2021-07-18 05:05:11,029] {python.py:151} INFO - Done. Returned value was: None
[2021-07-18 05:05:11,317] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=processing_tasks.task_2, execution_date=20210717T033923, start_date=20210718T050507, end_date=20210718T050511
[2021-07-18 05:05:11,992] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-18 05:05:12,025] {local_task_job.py:151} INFO - Task exited with return code 0
